packages/core/src/linker/component_factory_resolver.ts

```diff
@@ -40,6 +40,9 @@ class _NullComponentFactoryResolver implements ComponentFactoryResolver {
  * then use the factory's `create()` method to create a component of that type.
  *
  * @see [Dynamic Components](guide/dynamic-component-loader)
+ * @see [Usage Example](guide/dynamic-component-loader#resolving-components)
+ * @see <live-example name="dynamic-component-loader"></live-example>
```

Comment:
I think it would be nice to avoid the download link:

```suggestion
 * @see <live-example name="dynamic-component-loader" noDownload></live-example>
```

Reply:
Added.